static/app/views/organizationStats/usageStatsPerMin.tsx

```diff
@@ -52,17 +58,21 @@ function UsageStatsPerMin({dataCategory, organization, projectIds}: Props) {
     return null;
   }
 
+  const category =
+    dataCategoryApiName === 'span_indexed' ? dataCategoryApiName : dataCategory;
+
```

Comment:
also this shall be fine as we are filtering for 'accepted' only but just in case this corrects the category name.